src/components/FirstHeader.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { View, Text, StyleSheet } from "react-native";
+
+
+export const FirstHeader = () => (
+  <View style={styles.container}>
+      
+      <Text style={styles.main}>Todo</Text>
+      <Text style={styles.right}>+</Text>
+
+  </View>
+  
```

Comment:
```suggestion
  <View style={styles.container}>
      <Text style={styles.main}>Todo</Text>
      <Text style={styles.right}>+</Text>
  </View>
```

Don't you have **Prettier** configured along with `formatOnSave`?

Reply:
Already working, my bad